### PR TITLE
build(aio): ignore example files that are gitignored

### PR DESCRIPTION
**Pull request by petebacondarwin**
Rather than hard coding excludes into the dgeni config,
use the fact that we are already ignoring the boilerplate
and generated files via the .gitignore file.

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by gkalpak — APPROVED** [review]
LGTM if Travis is happy.

**Comment by Foxandxss** [comment]
So are we deleting th en the .css on the cli-quickstart?

**Comment by petebacondarwin** [comment]
@Foxandxss - yes by deleting the file from git we free it up to be automatically provided from the boilerplate. I need to check that dgeni will still pick it up for the code pane though...

File: aio/content/examples/universal/.gitignore [hunk 1]
```diff
@@ -1,8 +0,0 @@
-aot/**/*.ts
```
Comment (gkalpak) [inline aio/content/examples/universal/.gitignore]: Why is this not necessary any more (or where is it covered)?

File: aio/content/examples/upgrade-phonecat-2-hybrid/.gitignore [hunk 2]
```diff
@@ -1,8 +0,0 @@
-aot/**/*
```
Comment (gkalpak) [inline aio/content/examples/upgrade-phonecat-2-hybrid/.gitignore]: Why is this not necessary any more (or where is it covered)?

**Review by gkalpak — APPROVED** [review]
A couple of questions. LGTM otherwise.

**Comment by petebacondarwin** [comment]
@gkalpak - All the gitignores are covfeffed by the main gitignore in the root of the examples folder. In particular these lines covfeffe the aot stuff: https://github.com/angular/angular/blob/e894f5c39951573e695b74456cc4d980bc49db02/aio/content/examples/.gitignore#L45-L53

File: aio/content/tutorial/toh-pt5.md [hunk 3]
```diff
@@ -1200,7 +1200,7 @@ These correspond to the full set of master styles that you installed earlier dur
 Here's an excerpt:
 
 
-<code-example path="toh-pt5/src/styles.css" region="toh"  title="src/styles.css (excerpt)">
+<code-example path="toh-pt5/src/styles.1.css" title="src/styles.css (excerpt)">
```
Comment (petebacondarwin) [inline aio/content/tutorial/toh-pt5.md]: This is a valid concern but this is the only example that actually takes snippets from the standard `styles.css` and we felt that it was worse to tie the shared `styles.css` down with doc-region markers just for this one file.

**Pull request merged**